tests/e2escenarios/e2e_images_test.go

```diff
@@ -89,35 +89,35 @@ var _ = Describe("odo supported images e2e tests", func() {
 		})
 
 		It("Should be able to verify the nodejs-10 image", func() {
-			oc.ImportImageFromRegistry("registry.access.redhat.com", filepath.Join("rhoar-nodejs", "nodejs-10:latest"), "nodejs:latest", commonVar.Project)
-			verifySupportedImage(filepath.Join("rhoar-nodejs", "nodejs-10:latest"), "nodejs", "nodejs:latest", commonVar.Project, appName, commonVar.Context)
+			oc.ImportImageFromRegistry("registry.access.redhat.com", filepath.ToSlash(filepath.Join("rhoar-nodejs", "nodejs-10:latest")), "nodejs:latest", commonVar.Project)
```

Comment:
You should not treat this as a file path. This is what got us into this situation in the first place.
Your solution would technically work, but it is still wrong and problematic. 
Just providing the whole image name as a string is completely ok here (`rhoar-nodejs/nodejs-10:latest`). The image name will always have `/` as a separator.

Reply:
/hold 
please don't merge this before this gets properly fixed.

Reply:
Understood